From c5b7261cae14ada86f5823659664fade0bcefa20 Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Wed, 21 May 2014 19:43:30 +0200
Subject: [PATCH] ARM: dts: sun7i: cubietruck: set mmc3 bus-width property

bus-width defaults to 1, and all 4 lines are hooked up at the cubietruck,
properly set bus-width to 4.

Signed-off-by: Hans de Goede <hdegoede@redhat.com>
Signed-off-by: Maxime Ripard <maxime.ripard@free-electrons.com>
---
 arch/arm/boot/dts/sun7i-a20-cubietruck.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/sun7i-a20-cubietruck.dts b/arch/arm/boot/dts/sun7i-a20-cubietruck.dts
index b7e79d0..b87fea9 100644
--- a/arch/arm/boot/dts/sun7i-a20-cubietruck.dts
+++ b/arch/arm/boot/dts/sun7i-a20-cubietruck.dts
@@ -34,6 +34,7 @@
 			pinctrl-names = "default";
 			pinctrl-0 = <&mmc3_pins_a>;
 			vmmc-supply = <&reg_vmmc3>;
+			bus-width = <4>;
 			non-removable;
 			status = "okay";
 		};
-- 
1.9.0

